lxd/ip/link.go

```diff
@@ -77,6 +77,16 @@ func (l *Link) SetMTU(mtu string) error {
 	return nil
 }
 
+// SetTXQueueLength sets the txqueuelen of the link device.
+func (l *Link) SetTXQueueLength(txql string) error {
```

Comment:
Given that GetTXQueueLength returns a uint32, I think its more important for consistency that this function also accepts a uint32. Also can you name the argument `queueLength` or `length` rather than `txql` which is a little cryptic :)

Reply:
Done